config.py

```diff
@@ -283,8 +283,77 @@ def logging_policy(cls):
     def localization_languages(cls):
         languages = cls.policy(cls.LOCALIZATION_LANGUAGES, default=["eng"])
         return [LanguageCodes.three_to_two[l] for l in languages]
+
+    # The last time the database configuration is known to have changed.
+    SITE_CONFIGURATION_LAST_UPDATE = "site_configuration_last_update"
```

Comment:
My goal is to not have to go to the database to check this more than once a minute. I was thinking I would put that code in circulation but maybe I got confused and it needs to go into core. I'll investigate.